modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing an upstream vSphere CSI Operator Driver
+
+If you have installed an upstream vSphere Container Storage Interface (CSI) Operator Driver, and then try to update your cluster to {product-title} 4.10 or later, {product-title} prevents the update.
+
+To correct this issue and update the cluster, a cluster admin must:
```

Comment:
We should also mention that, removing upstream CSI driver does not require deletion of associated PV objects and no data loss should happen.